.gitlab-ci.yml

```diff
@@ -59,30 +59,23 @@ stages:
     - echo ${ALLOC_NAME}
     - export JOBID=$(squeue -h --name=${ALLOC_NAME} --format=%A)
     - echo ${JOBID}
-    - srun $( [[ -n "${JOBID}" ]] && echo "--jobid=${JOBID}" ) -t 10 -N 1 -n 1 -c 4 scripts/gitlab/build_and_test.sh --build-only
-
-.test_toss_3_x86_64_ib_script:
-  script:
-    - echo ${ALLOC_NAME}
-    - export JOBID=$(squeue -h --name=${ALLOC_NAME} --format=%A)
-    - echo ${JOBID}
-    - srun $( [[ -n "${JOBID}" ]] && echo "--jobid=${JOBID}" ) -t 10 -N 1 -n 1 -c 4 scripts/gitlab/build_and_test.sh --test-only
+    - srun $( [[ -n "${JOBID}" ]] && echo "--jobid=${JOBID}" ) -t 10 -N 1 scripts/gitlab/build_and_test.sh
```

Comment:
Will this result in a single failure for either build or test?  One of the things I would like to see is a (more) clear indication of whether the failure is due to an infrastructure (git failures), build, or test failure.  Idealy this would be clearly shown in the subject line of the email.

Reply:
We could surely improve that. Please open an issue describing what you would like to have, that’s the best way for me to keep track of tasks.

Reply:
#381